src/cephadm/cephadm

```diff
@@ -5561,6 +5692,14 @@ def command_gather_facts():
 
 ##################################
 
+def command_verify_prereqs():
+    if args.service_type == 'haproxy' or args.service_type == 'keepalived':
+        out, err, code = call(['sysctl', '-n', 'net.ipv4.ip_nonlocal_bind'])
+        if out.strip() != "1":
+            raise Error('net.ipv4.ip_nonlocal_bind not set to 1')
```

Comment:
there is a big uxer experience problem here:

1. `ceph orch apply rgw-ha --placement count:2`
2. `HostAssignmenet` chooses host1 and host2
3. we're calling `deploy_daemon('host1')`
4. `deploy_daemon` calls `command_verify_prereqs`
5. `command_verify_prereqs` fails. 
6. at that point, we can't do anything about it. 

I'd recommend that we make sure, we have all the information  necessary to schedule daemons to cephadm's scheduler. I'd expose `net.ipv4.ip_nonlocal_bind` to 

https://github.com/ceph/ceph/blob/ba99f56f52c01c7bea1ddd677746150ea1265ee0/src/pybind/mgr/cephadm/serve.py#L107

and then enable us to make sensible scheduling decisions.